x-pack/functionbeat/docs/getting-started.asciidoc

```diff
@@ -0,0 +1,201 @@
+https://www.elastic.co/downloads/beats/{beatname_lc}[downloads page].
+
+. Extract the contents of the zip file. 
+
+QUESTION: Is powershell required to run the deploy command?
+
+endif::[]
+
+[id="{beatname_lc}-configuration"]
+=== Step 2: Configure {beatname_uc}
+
+Before deploying {beatname_uc} to your serverless environment, you need to
+specify details about the functions that you want to deploy, including the
+function name, type, and triggers that will invoke the function. You also need
+to specify connection details for your {es} cluster. 
+
+You specify settings in the  +{beatname_lc}.yml+ configuration file. This file
+is located in the archive that you extracted earlier.
+
+TIP: See the
+{libbeat}/config-file-format.html[Config File Format] section of the
+_Beats Platform Reference_ for more about the structure of the config file.
+
+The following example configures a function called `myfunction` that collects
+events from Cloudwatch Logs and forwards the events to {es}.
+
+["source","sh",subs="attributes"]
+-------------------------------------------------------------------------------------
+ADD EXAMPLE HERE.
+-------------------------------------------------------------------------------------
+
+To configure {beatname_uc}:
+
+. Define the functions that will execute in your serverless environment.
++
+TODO: Explain what is required (name, type?, one or more triggers, etc. Will
+need to desribe triggers somewhere else maybe and refer to those?
++
+["source","sh",subs="attributes"]
+-------------------------------------------------------------------------------------
+ADD AN EXAMPLE IF NEEDED
+-------------------------------------------------------------------------------------
+
+include::{libbeat-dir}/docs/step-configure-output.asciidoc[]
+
+include::{libbeat-dir}/docs/step-configure-kibana-endpoint.asciidoc[]
+
+include::{libbeat-dir}/docs/step-configure-credentials.asciidoc[]
+
+include::{libbeat-dir}/docs/step-test-config.asciidoc[]
+
+include::{libbeat-dir}/docs/step-look-at-config.asciidoc[]
+
+[id="{beatname_lc}-template"]
+=== Step 3: Load the index template in Elasticsearch
+
+:allplatforms:
+include::{libbeat-dir}/docs/shared-template-load.asciidoc[]
+
+[[load-kibana-dashboards]]
+=== Step 4: Set up the Kibana dashboards
+
+:allplatforms:
+include::{libbeat-dir}/docs/dashboards.asciidoc[]
+
+[id="{beatname_lc}-starting"]
+=== Step 5: Deploy {beatname_uc} code to your serverless environment
+
+Before deploying functions to your serverless environment, make sure you're
+logged in with the credentials required by your cloud service provider. 
+
+QUESTION: Does the user need specific privileges to run functionbeat?
+
+The following command deploys the function named `myfunction`. See xxx for more
+information about the `deploy` command.
+
+TODO: Add link that points to the command reference for more information.
+
+QUESTION: Which flags do we want to show in the example here? debug selectors?
```

Comment:
@ph Which selector should we show? TBH, I can't remember them all because we decided not to document them. I've only used -d "publish", or (when I'm feeling wild) -d "*"

Reply:
Closing because I've updated the section about selectors.